.github/pull_request_template.md

```diff
@@ -14,10 +14,19 @@ and code of conduct https://github.com/grafana/k6/blob/master/CODE_OF_CONDUCT.md
 -->
 
 - [ ] I have performed a self-review of my code.
-- [ ] I have added tests for my changes.
-- [ ] I have run linter locally (`make lint`) and all checks pass.
-- [ ] I have run tests locally (`make tests`) and all tests pass.
 - [ ] I have commented on my code, particularly in hard-to-understand areas.
+- [ ] I have added tests for my changes.
+- [ ] I have run linter and tests locally (`make lint` and `make tests`) and all pass.
+
+## Checklist: Documentation (only for k6 maintainers and if relevant)
+
+**Please do not merge this PR until the following items are filled out.**
+
+- [ ] I have added the correct milestone to the PR.
+- [ ] I have updated the k6-documentation: _link_
+- [ ] I have updated the TypeScript definitions: _link_
+- [ ] I have updated the release notes: _link_
```

Comment:
Maybe link to the repositories, not expected, but it could foster contributions there

```suggestion
- [ ] I have updated [the k6-documentation](https://github.com/grafana/k6-docs): _link_
- [ ] I have updated the [TypeScript definitions](https://github.com/grafana/k6-DefinitelyTyped/tree/master/types/k6): _link_
- [ ] I have updated the release notes: _link_
```